packages/mdc-textfield/foundation.js

```diff
@@ -99,7 +100,7 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
   }
 
   deactivateFocus_() {
-    const {FOCUSED, INVALID, LABEL_FLOAT_ABOVE} = MDCTextfieldFoundation.cssClasses;
+    const {FOCUSED, LABEL_FLOAT_ABOVE} = MDCTextfieldFoundation.cssClasses;
     const input = this.getNativeInput_();
     const isValid = input.checkValidity();
```

Comment:
I will move the input check in the if block just for performance reasons like so:
`if (!this.useCustomValidityChecking_) {
  this.changeValidity_(input.checkValidity());
}`